appinfo/info.xml

```diff
@@ -7,7 +7,7 @@
 	<description><![CDATA[Push update support for desktop app.
 
 Once the app is installed, the push binary needs to be setup. You can either use the setup wizard with `occ notify_push:setup` or see the [README](http://github.com/nextcloud/notify_push) for detailed setup instructions]]></description>
-	<version>0.3.0</version>
+	<version>0.3.1</version>
```

Comment:
Bumping the version should probably be done as part of the release process keeping the purpose of this commit (allowing NC24) to be separate.

Reply:
Agreed, reverted the app version change (0.3.1→0.3.0).